test/studies/ssca2/performance/SSCA2_commDiags.prediff

```diff
@@ -47,7 +47,7 @@ for l in lines:
             for d in range(10):
                 target = counts[i][d]
                 curr=int(commDiags[d].split('=')[1].strip())
-                if d==0 or d==7:
+                if d==0 or d==7 or d==8:
```

Comment:
This test previously filtered out the number of executeOns by replacing that number with N/A. Since some of those changed to executeOnFast, and the number is still nondeterministic, I filtered out the executeOnFasts too. You can see it in the .good file update: the fork_fast lines now say N/A instead of 0. In fact the actual number is nonzero.